nvflare/private/fed/client/training_cmds.py

```diff
@@ -118,12 +118,6 @@ def process(self, req: Message, app_ctx) -> Message:
         if not job_meta:
             return error_reply("missing job meta")
 
-        err = engine.deploy_app(
```

Comment:
I already fixed this in my branch. Please pull again from my branch.

Reply:
ok. rebased on your branch.